api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -1059,21 +1059,12 @@ public static String getApplicationDataDirectory() {
 					filepath = OpenmrsConstants.APPLICATION_DATA_DIRECTORY_FALLBACK_UNIX + File.separator + "OpenMRS";
 				}
 			} else {
-				if (OpenmrsConstants.UNIX_BASED_OPERATING_SYSTEM) {
-					filepath = System.getProperty("user.home") + File.separator + ".OpenMRS";
-					if (!canWrite(new File(filepath))) {
-						log.warn("Unable to write to users home dir, fallback to: "
-						        + OpenmrsConstants.APPLICATION_DATA_DIRECTORY_FALLBACK_UNIX);
-						filepath = OpenmrsConstants.APPLICATION_DATA_DIRECTORY_FALLBACK_UNIX + File.separator + "OpenMRS";
-					}
-				} else {
-					filepath = System.getProperty("user.home") + File.separator + "Application Data" + File.separator
-					        + "OpenMRS";
-					if (!canWrite(new File(filepath))) {
-						log.warn("Unable to write to users home dir, fallback to: "
-						        + OpenmrsConstants.APPLICATION_DATA_DIRECTORY_FALLBACK_WIN);
-						filepath = OpenmrsConstants.APPLICATION_DATA_DIRECTORY_FALLBACK_WIN + File.separator + "OpenMRS";
-					}
+				filepath = System.getProperty("user.home") + File.separator + "Application Data" + File.separator
+						+ "OpenMRS";
```

Comment:
The duplication was https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/util/OpenmrsUtil.java#L1055-L1069 and not what you removed.